NOnion/Cells/CellCreated2.fs

```diff
@@ -0,0 +1,34 @@
+﻿namespace NOnion.Cells
+
+open System.IO
+
+open NOnion.Utility
+
+type CellCreated2 =
```

Comment:
That's the name of the cell in spec and we should be inline with it, There is a create cell as well but I didn't implement it yet because it's legacy and only for backward compatibility with old routers